src/deprecated/admin/class-inclusive-language-notice.php

```diff
@@ -49,6 +56,9 @@ public function __construct( Yoast_Notification_Center $notification_center ) {
 
```

Comment:
I decided not to replace the content of the functions with `_deprecated_function( __METHOD__, 'WPSEO 19.7' )` in order to preserve their content. This approach has been used for other deprecated classes, such as `src/deprecated/admin/keyword-synonyms-modal.php`. 
Hopefully this works 🤞

Reply:
We can do both. Add `_deprecated_function` (e.g. at the top of the function body) and leave the existing code intact otherwise.

That has the advantage that whenever a developer uses this code, they get a warning in WordPress itself (https://developer.wordpress.org/reference/functions/_deprecated_function/).